types/v1/k8s_storage_class.go

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Can this file be renamed to -- types/volumepolicy/v1.go ?
This is generic stuff, just happens to be provided via SC in case of integration via K8s.

Reply:
Agreed with types/policies/
However, should it be v1.go or v1/policies.go

Reply:
either is fine. as long as we stick to one convention.